src/Circulator.sol

```diff
@@ -222,14 +266,41 @@ contract Circulator is ICirculator, FeeOperator, Pausable, EIP712, Nonces {
         _fee = (_amount * serviceFeeBPS) / 1e4;
     }
 
+    /**
+     * @notice Initializes the destination domain configurations.
+     * @dev Only callable by the contract owner.
+     * @param _domainIds List of domain IDs.
+     * @param _relayerFees List of relayer fees corresponding to each domain ID.
+     * @param _minFees List of base fees corresponding to each domain ID.
+     * @param _chainIds List of chain IDs corresponding to each domain ID.
+     * @param _tokens List of token addresses corresponding to each domain ID.
+     */
+    function initDestinationConfigs(
+        uint32[] memory _domainIds,
+        uint256[] memory _relayerFees,
+        uint256[] memory _minFees,
+        uint256[] memory _chainIds,
+        address[] memory _tokens
+    ) external onlyOwner {
```

Comment:
we should add `Initializable` + `initialiser` modifier to make sure it can only be called once